examples/collector-agent-host-network-disabled/README.md

```diff
@@ -0,0 +1,8 @@
+# Example of chart configuration
+
+## Deploy the Splunk Collector in agent mode and disable using the host network
+
+- This configuration will install the collector as an agent deployment only.
+- Disabling this value will affect monitoring of some control plane components.
+- Enabling the agent service is recommended and also done in this example. Kubernetes services will be deployed to use transmit collector related data.
```

Comment:
I'm not sure about the wording here, but, making a suggestion to start a conversation about it:
```suggestion
- Enabling the agent service, as done in this example, is recommended. In this case Kubernetes services will be deployed to transmit collector related data.
```